<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>DialogProb</class>
  <widget class="QDialog" name="dialog_prob">
    <property name="windowTitle">
      <string>Oxidation Probabilities</string>
    </property>
    <property name="modal">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
              <widget class="QLabel" name="label">
                <property name="text">
                  <string>OH Probability (%):</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="QSpinBox" name="spin_prob_oh">
                <property name="minimum">
                  <number>0</number>
                </property>
                <property name="maximum">
                  <number>100</number>
                </property>
                <property name="value">
                  <number>66</number>
                </property>
                <property name="singleStep">
                  <number>1</number>
                </property>
              </widget>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="label_2">
                <property name="text">
                  <string>O Probability (%):</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
              </widget>
            </item>
            <item row="1" column="1">
              <widget class="QSpinBox" name="spin_prob_o">
                <property name="minimum">
                  <number>0</number>
                </property>
                <property name="maximum">
                  <number>100</number>
                </property>
                <property name="value">
                  <number>34</number>
                </property>
                <property name="singleStep">
                  <number>5</number>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QDialogButtonBox" name="buttonBox">
          <property name="orientation">
            <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections>
    <connection>
      <sender>buttonBox</sender>
      <signal>accepted()</signal>
      <receiver>dialog_prob</receiver>
      <slot>accept()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>248</x>
          <y>254</y>
        </hint>
        <hint type="destinationlabel">
          <x>157</x>
          <y>274</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>buttonBox</sender>
      <signal>rejected()</signal>
      <receiver>dialog_prob</receiver>
      <slot>reject()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>316</x>
          <y>260</y>
        </hint>
        <hint type="destinationlabel">
          <x>286</x>
          <y>274</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>
